packages/app/translations/en.json

```diff
@@ -0,0 +1,75 @@
+{
+    "general": {
+        "title": "Open school platform",
+        "loading": "Loading...",
+        "logout": "Logout",
+        "abort": "Abort",
+        "settings": "Settings",
+        "send": "Send",
+        "confirm": "Confirm",
+        "socialSecurityNumber": "Personal number"
+    },
+    "navigation": {
+        "news": "News",
+        "notifications": "Notifications",
+        "calender": "Calender",
+        "menu": "Menu"
+    },
+    "auth": {
+        "subtitle": "The {{word}} alternative",
+        "personalNumber": "Personal number",
+        "bankid_OpenOnAnotherDevice": "Open BankID on another device",
+        "bankid_OpenOnThisDevice": "Open BankID on this device",
+        "bankid_Waiting": "Waiting for BankID...",
+        "bankid_OpenManually": "Open BankID manually",
```

Comment:
Would it make sense to group these under a common `"bankid": {...}`, so that references don't need to mix `.` and `_`? Turning references to `'bankid_Waiting'`->`'bankid.waiting'`, etc. 
Then again, this might increase complexity for potential translators, as a trade-off for the grouping.

Reply:
Yea, think we need context, and as you say its a trade off, not sure how we are going tackle the translations at all but :D

Reply:
My humble vote is for using the `'x.y.z'` principle, thus grouping them in their own common object - at least I imagine it could end up decreasing potential copy+paste errors as well as signalling where new strings should be added. 👼

Reply:
I've fixed that now! Good idea